include/seqan3/argument_parser/detail/format_base.hpp

```diff
@@ -283,14 +293,22 @@ class format_help_base : public format_base
      *
      * \param[out] value     The variable in which to store the given command line argument.
      * \param[in]  desc      The description of the positional option.
-     * \param[in]  validator The validator applied to the value after parsing (callable).
+     * \param[in]  val       The validator applied to the value after parsing (callable).
```

Comment:
... from the person who criticizes me for short variable names? :laughing:

Reply:
BTW. This is an error my doxygen warns me about but that CI didn't catch.

Reply:
Well, I think having a parameter called `value` and having one called `val` (which is the typical abbreviation of value) is quite confusing to be honest...